README.md

```diff
@@ -17,6 +17,7 @@
 	 -i	Set timeout value for project in seconds. Will use 600s (10min) unless specified
 	 -f	Set fail threshold in percentage [0-100], the percentage of test steps that have to pass for the test run to succeed (impacts exit value)
 	 -x	Set device completion threshold in percentage [0-100], the percentage of devices in device group that need to complete for the test run to complete (impacts exit value)
+	 -n	Specify a test_run_id, client will only fetch those results and exit
```

Comment:
No, this is what they call testRunId and runId in testdroid API.
http://docs.testdroid.com/testdroid-cloud-integration/api/rest-api/

It is true that it is confusing that the id is being referred to here for the first time. Should we include the API-link in the README.md or could there be a better way explaining the id?